### PR TITLE
add no results found

### LINKED ISSUE #24: Add a "No Results" view

### PR DESCRIPTION
**Pull request by dandashino**
Added a very simple not found view to resolve #24. Of course this can be later on replaced with any other more visually appealing view.

### CODE REVIEW

File: web/src/views/Home.scss [hunk 1]
```diff
@@ -28,3 +28,11 @@
 .recent-searches {
   display: flex;
 }
+
+.no-results {
+  text-align: center;
+}
+
+.no-results h1 {
+  font-size:580%;
```
Comment (mrahhal) [inline web/src/views/Home.scss]: nit: space between the colon and the value.

File: web/src/views/Home.scss [hunk 2]
```diff
@@ -28,3 +28,11 @@
...
+}
```
Comment (mrahhal) [inline web/src/views/Home.scss]: nit: all files end with an empty new line.

File: web/src/views/Home.vue [hunk 3]
```diff
@@ -65,17 +65,24 @@
...
+    <template v-else-if="noResultsFound">
+      <div class="no-results">
+        <h1>無</h1>
+        <h3>No results found</h3>
+      </div>
+    </template>
+
     <div class="text-2" v-else>
       Just start typing to search for similar Kanjis, homonyms, and synonyms!
-      <br>
```
Comment (mrahhal) [inline web/src/views/Home.vue]: Shouldn't edit those and what follows.

File: web/src/views/Home.scss (hunk 1 above)
Comment (dandashino) [reply web/src/views/Home.scss]: > Thanks for this! Just a minor edit: please keep `br`/`hr` elements the way they were (in "Home.vue"). `<br>` is sufficient in html and need those to be consistent across all files.

Oh sorry thought i ignored that change, I ran lint by mistake so it did some changes.

**Comment by mrahhal** [comment]
Thank you! Changes will be deployed by today.

**Pull request merged**